spec/shr_oncologycore.txt

```diff
@@ -42,151 +42,167 @@ Concept:        LNC#21858-6 "Grade Cancer"
+    0..0        FindingResult  // true panel
+    0..0        NonIndependentFinding
     0..0        ExceptionValue
     0..0        ReferenceRange
     0..0		AnatomicalLocation
     0..0        Specimen
     0..0        Device
-    0..1        FindingMethod from CancerStagingSystemVS
-                SpecificFocusOfFinding value is type ref(CancerDisorderPresent)
-    NonIndependentFinding
-        includes  0..1 StageGroup  // MK - should this be the FindingResult?
-    PanelMembers.Observation
-        includes  0..1 TNMClinicalPrimaryTumorClassification
-        includes  0..1 TNMClinicalRegionalNodesClassification
-        includes  0..1 TNMClinicalDistantMetastasesClassification
-        includes  0..1 TNMPathologicPrimaryTumorClassification
-        includes  0..1 TNMPathologicRegionalNodesClassification
-        includes  0..1 TNMPathologicDistantMetastasesClassification
-
-Element:        StageTiming
+    0..0        FindingMethod // The expectation is that each member of the panel will declares its own method; if added here, it would be redundant.
+
+
+EntryElement:   TNMClinicalStagePanel
+Based on:       CancerStagePanel
+Concept:        SCT#385349001   "Clinical stage (observable entity)"
+Description:    "Clinical staging is performed prior to surgery, based on evidence such as physical examination, imaging, and/or biopsy."
+                FindingTopicCode is SCT#385349001 "Clinical stage (observable entity)"
+                PanelMembers.Observation
+                    includes  0..1 TNMClinicalStageGroup
+                    includes  0..1 TNMClinicalPrimaryTumorClassification
+                    includes  0..1 TNMClinicalRegionalNodesClassification
+                    includes  0..1 TNMClinicalDistantMetastasesClassification
+
+EntryElement:   TNMPathologicStagePanel
+Based on:       CancerStagePanel
+Concept:        SCT#405978005   "Pathologic stage (observable entity)"
+Description:    "Pathologic stage is based on examination of tissue samples removed during surgery, in addition to physical examination and imaging."
+                FindingTopicCode is SCT#405978005   "Pathologic stage (observable entity)"
+                PanelMembers.Observation
+                    includes  0..1 TNMPathologicStageGroup
+                    includes  0..1 TNMPathologicPrimaryTumorClassification
+                    includes  0..1 TNMPathologicRegionalNodesClassification
+                    includes  0..1 TNMPathologicDistantMetastasesClassification
+
+
+Element:        CancerStageTiming 
 Concept:        SCT#260869008 "Timing of stage"
 Description:	"Indicates when the staging was done, in terms of treatment landmarks."
 Value:          CodeableConcept from StageTimingPrefixVS
 
-Element:        StageGroup
-// did not use precoordinated codes because only available for clinical and pathologic.
-Based on:       CodedNonIndependentFinding
+
+Abstract Element: TNMStageGroup
```

Comment:
Should be `StageGroup`. Stage does not necessary follow the TNM system.

Reply:
We are including the stage group in the TNM panel. It should be a TNM stage group.

Reply:
That pragmatically makes sense if we are to group all TNM staging components in one instance.  The downside is that there now needs to be a separate element for StageGroup to address non-TNM staging systems that just have a stage group?  I mentioned Lugano (above comment), but there's also Rai, Binet, R-ISS, etc.